src/main/java/com/spotinst/sdkjava/model/bl/aws/managed/instance/AmiBackup.java

```diff
@@ -0,0 +1,68 @@
+package com.spotinst.sdkjava.model.bl.aws.managed.instance;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class AmiBackup {
```

Comment:
Didn't find this class in the documentation, need discussion.

Reply:
this is for deleteManagedInstance api